dataset-conversion/convert_bids_to_nnUNetv2_all_sci_data.py

```diff
@@ -64,40 +65,32 @@ def get_parser():
     parser.add_argument('--split', nargs='+', required=True, type=float, default=[0.8, 0.2],
                         help='Ratios of training (includes validation) and test splits lying between 0-1. Example: '
                              '--split 0.8 0.2')
+    # input yaml file containing list of axial subjects to include for active learning
+    parser.add_argument('--include-axial', type=str, default=None,
+                        help='Path to yaml file containing list of axial subjects to include for active learning.')
```

Comment:
How is this YAML file obtained? Downloaded from HTML QC or created manually?

Reply:
I would say it is created semi-manually. The HTML QC also contains info such as ses-xx, contrast, etc. But this yaml file simply contains the subject names to be included. Example for a few subjects:

```
CORR_LESION:
- sub-zh01
- sub-zh03
- sub-zh05
- sub-zh10
- sub-zh14
- sub-zh25
- sub-zh30
- sub-zh32
- sub-zh33
- sub-zh34
- sub-zh38
- sub-zh41
- sub-zh42
```

Reply:
Okay!